runner/engine_test.go

```diff
@@ -895,11 +902,11 @@ include_ext = ["sh"]
 include_dir = ["nonexist"] # prevent default "." watch from taking effect
 include_file = ["main.sh"]
 `
-	if err := ioutil.WriteFile(dftTOML, []byte(config), 0644); err != nil {
+	if err := ioutil.WriteFile(dftTOML, []byte(config), 0o644); err != nil {
```

Comment:
Is there a need to change?

Reply:
Again, just let me know if you prefer the PR without this...

Reply:
OK